odsApi_versioned_docs/version-7.3/whats-new/release-notes.md

```diff
@@ -130,3 +135,12 @@ Depending on the configuration of the deployed ODS, SchoolYear segment may not
 be present or in case when additional segments are present, they appear at the
 beginning of all routes allowing easy configuration for API clients. See [API
 Routes](../client-developers-guide/api-routes.md) for details.
+
+#### Error Responses
+
+Throughout the API surface: Starting with version 7.3, ODS/API updated its error
+responses to implement [Problem Details RFC
+9457](https://www.rfc-editor.org/rfc/rfc9457.html), providing both
+machine-readable and user-friendly responses. This change may require
+modifications to API Client applications. See [Error Response Knowledge
+Base](../client-developers-guide/error-response-knowledge-base.md) for details.
```

Comment:
Should this paragraph be removed, since the problem details format was introduced in the prior release?

Reply:
This is listing all breaking changes for folks who are looking for implementing DS 4 with API 7.x